Tribler/Test/Core/test_sqlitecachedb.py

```diff
@@ -35,13 +35,20 @@ def tearDown(self):
         self.sqlite_test = None
 
     @blocking_call_on_reactor_thread
-    def test_create_db(self):
+    def test_create_person_db(self):
```

Comment:
Minor detail: I think it's better to leave the name as it was before because this test is actually testing whether any database can be created, not that a `person` database in particular can be created.